src/mpi/request/testsome.c

```diff
@@ -31,11 +31,53 @@ int MPI_Testsome(int incount, MPI_Request array_of_requests[], int *outcount,
 #undef MPI_Testsome
 #define MPI_Testsome PMPI_Testsome
 
+#undef FUNCNAME
```

Comment:
I'm not sure if I authored this patch. Perhaps only Michael. According to #2742 I did not.

Reply:
fixed